src/test/resources/expected-classes/request/graphql-query/eventsByIdsQuery.txt

```diff
@@ -1,5 +1,5 @@
 query {
-  eventsByIds(ids: [ "4", "5", "6" ]){
+  eventsByIds(ids: [ \"\u1234\", \"\"\", \"\n\" ]){
```

Comment:
I believe here the query should not escape the outermost quotes of each string, otherwise it's probably invalid from a GraphQL standpoint. We expect `ids` to be of type `[String]` here, so we should really provide a list of double-quoted strings.

Reply:
@joffrey-bion Fixed. Thank you!